src/economy/economy.cpp

```diff
@@ -3415,11 +3877,25 @@ void daily_update(sys::state& state) {
 	// make constructions:
 	resolve_constructions(state);
 
+	if(!initiate_buildings) {
+		goto AFTER_INVESTMENTS;
```

Comment:
just write the if block to surround the entire section, as using a goto will set off many linters